### PR TITLE
Add multiple pods with anti affinity

### PR DESCRIPTION
**Pull request by EliiseS**
This PR is to address the issue [#164 Operator becomes unavailable if pod restarts](#164). We've increased the operator pod count to 2 and the pods will be deployed on different nodes to keep the operator operational in case a pod or a node goes down.

We've also tested that this change has no negative effect on the performance of the operator, the results of the test can be seen below:
image
image

### CODE REVIEW

File: config/manager/manager.yaml [hunk 1]
```diff
@@ -16,12 +16,22 @@ spec:
...
-  replicas: 1
+  replicas: 2
   template:
     metadata:
       labels:
         control-plane: controller-manager
     spec:
+      affinity:
+          podAntiAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
```
Comment (lawrencegripper) [inline config/manager/manager.yaml]: Thought: Would it be better to go with `preferredDuringSchedulingIgnoredDuringExecution` so that 2 pods would still be created even if the operator was being used on a single node cluster. 

>There are currently two types of node affinity, called requiredDuringSchedulingIgnoredDuringExecution and preferredDuringSchedulingIgnoredDuringExecution. You can think of them as “hard” and “soft” respectively, in the sense that the former specifies rules that must be met for a pod to be scheduled onto a node (just like nodeSelector but using a more expressive syntax), while the latter specifies preferences that the scheduler will try to enforce but will not guarantee.

https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#node-affinity

Comment (EliiseS) [reply config/manager/manager.yaml]: I think that's a good point. I was considering this myself and you've pushed me over the edge
